@mjcarroll  Hi michael, I m facing an error msg in the CI test, in which I have no idea why. :thinking:  

error msg:
```bash
=========================== short test summary info ============================
FAILED test/test_build_caches.py::test_build_caches - RuntimeError: noetic: missing package.xml file for package "serl_franka_controllers"
======== 1 failed, 6 passed, 2 skipped, 3 warnings in 65.32s (0:01:05) =========
```

My `package.xml` is avail in https://github.com/rail-berkeley/serl_franka_controllers. Thanks for your help

--- a/noetic/distribution.yaml
+++ b/noetic/distribution.yaml
@@ -11082,16 +11082,11 @@ repositories:
   serl_franka_controllers:
     doc:
       type: git
-      url: https://github.com//rail-berkeley/serl_franka_controllers.git
+      url: https://github.com/rail-berkeley/serl_franka_controllers.git
       version: main
-    release:
-      tags:
-        release: release/noetic/{package}/{version}
-      url: https://github.com//rail-berkeley/serl_franka_controllers.git
-      version: 0.1.0-1
     source:
       type: git
-      url: https://github.com//rail-berkeley/serl_franka_controllers.git
+      url: https://github.com/rail-berkeley/serl_franka_controllers.git
       version: main
     status: maintained
   sick_ldmrs_laser:

